validation/cross_validate_rust_python.py

```
@@ -102,11 +118,25 @@ def carreau_yasuda_python(shear_rate):
+
+    for gamma_dot in test_shear_rates:
+        mu_python = carreau_yasuda_python(gamma_dot)
+        mu_rust = blood.apparent_viscosity(gamma_dot)
+        diff_pct = abs(mu_python - mu_rust) / mu_python * 100
+
+        print(f"{gamma_dot:20.0f} {mu_rust*1000:15.4f} {diff_pct:14.6f}%")
+        if diff_pct >= 1e-4:
+            passed = False
+
+    if passed:
+        print(f"  ✓ Cross-validation passed")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)